src/components/BreatheAnimation/BreatheAnimation.js

```diff
@@ -0,0 +1,67 @@
+import styled, {keyframes} from 'styled-components';
+
+export default function BreatheAnimation() {
+  return (
+    <Container>
+      <Circle />
+      <StyledParagraphIn>Ein</StyledParagraphIn>
+      <StyledParagraphOut>Aus</StyledParagraphOut>
```

Comment:
I am wondering whether a `span` element is more semantic than a paragraph, because you have only one word inside of it.